Cli/Extensions.lean

```diff
@@ -38,7 +44,7 @@ section Utils
   end Array
 end Utils
 
-section Extensions
+public section Extensions
```

Comment:
I think we may actually also want to expose the Utils above, since I suspect they are often useful for downstream extensions as well.

Reply:
You made the section in `Basic` `public` :-)